Stripe/STPRedirectContext.m

```diff
@@ -194,30 +245,23 @@ - (void)handleRedirectCompletionWithError:(nullable NSError *)error
 }
 
 - (void)subscribeToUrlAndForegroundNotifications {
-    [[NSNotificationCenter defaultCenter] addObserver:self
-                                             selector:@selector(handleWillForegroundNotification)
-                                                 name:UIApplicationWillEnterForegroundNotification
-                                               object:nil];
-    [[STPURLCallbackHandler shared] registerListener:self
-                                              forURL:self.source.redirect.returnURL];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleApplicationWillEnterForegroundNotification) name:UIApplicationWillEnterForegroundNotification object:nil];
```

Comment:
Why did you go through and remove the line breaks from like, every method call in all these files? I find it much harder to read like this.